### ISSUE TITLE
Photo Batch not loading

### ISSUE THREAD

**Issue by Flam3z** [oxen-io/session-android#751]
**Code of conduct**

- [x] I have read and agree to adhere to the [Code of Conduct](https://github.com/oxen-io/session-android/blob/master/CODE_OF_CONDUCT.md).

**Describe the bug**
A user in a closed group had sent a semi-big batch of photos and only 25% of them actually loaded while the others stay on a black screen while the loading animation continues even after 10 minutes.

**Smartphone (please complete the following information):**

 - Device: Galaxy S10
 - OS: Android 11
 - Version: 1.11.10

**Comment by KeeJef** [comment]
Is this continuing to occur?

**Comment by Flam3z** [comment]
Yes and of course after the 14 day period i cant load the images anyways.
